docs/update_release_notes.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
```

Comment:
Instead of this file being in `docs/`, let's create a 'scripts' directory for things like this.